package/autoyast2.spec

```diff
@@ -69,8 +70,6 @@ Obsoletes:      yast2-module-autoinst
 Provides:       yast2-lib-autoinst
 Obsoletes:      yast2-lib-autoinst
 
-PreReq:         %insserv_prereq %fillup_prereq
-
```

Comment:
this is related to drop of insserv. I think it is no longer used, as SLE 12 use systemd.